production/loki-mixin/dashboards/loki-deletion.libsonnet

```diff
@@ -2,7 +2,7 @@ local g = import 'grafana-builder/grafana.libsonnet';
 local utils = import 'mixin-utils/utils.libsonnet';
 
 (import 'dashboard-utils.libsonnet') {
-  local compactor_matcher = 'pod=~"(compactor|%s-backend.*|loki-single-binary)"' % $._config.ssd.pod_prefix_matcher,
+  local compactor_matcher = 'pod=~"(%scompactor.*|%s-backend.*|loki-single-binary)"' % [$._config.pod_prefix_matcher, $._config.ssd.pod_prefix_matcher],
```

Comment:
I also added a missing `.*` in this selector to the end of compactor, which is why the generated dashboards above have changed. I believe this was a defect - see the equivalent matcher in loki-retention.libsonnet which already has the `.*` (https://github.com/grafana/loki/blob/51c42e864563f2fa9ffc160cb13f6d6126ea5c6d/production/loki-mixin/dashboards/loki-retention.libsonnet#L3).